src/stylesheets/screens/survey-detail.scss

```diff
@@ -0,0 +1,55 @@
+.survey-detail {
+  width: 100vw;
+  height: 100vh;
+
+  &__header {
+    @include header-container();
+
+    padding: rem(16px 16px 0);
+  }
+
+  &__header-left {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__description {
+    @include details-container();
+    position: relative;
+
+    flex: 1;
+  }
+
+  &__image {
+    width: 100%;
+    height: rem(302px);
+    margin-top: rem(55px);
+
+    border-radius: rem(12px);
+  }
+
+  &__title {
+    margin-top: rem(20px);
+
+    color: map-get($brand-white, 'primary');
+    font-weight: map-get($font-weight, 'bolder');
+    font-size: map-get($font-sizes, 'xxxl');
+    line-height: rem(28px);
+  }
+
+  &__subtitle {
+    margin-top: rem(8px);
+
+    color: map-get($brand-white, 'primary');
+    font-weight: map-get($font-weight, 'normal');
+    font-size: map-get($font-sizes, 'lg');
+    line-height: rem(22px);
+
+    opacity: 0.6;
+  }
+
+  &__start-survey {
+    margin-top: rem(20px);
```

Comment:
@rafayet-monon this also can we remove margin-bottom from `survey-detail__subtitle` and use `rem(32px)` here?
![Screen Shot 2563-11-24 at 17 40 40](https://user-images.githubusercontent.com/1772999/100083593-3c742880-2e7c-11eb-9b9c-0e043fc2dce7.png)

![Screen Shot 2563-11-24 at 17 39 00](https://user-images.githubusercontent.com/1772999/100083431-0fc01100-2e7c-11eb-8d7d-5a606ad0c11a.png)